src/services/connect.ts

```diff
@@ -0,0 +1,36 @@
+import { StellarSDK } from '../api/stellarSdk';
+import type { Keypair } from 'stellar-sdk';
+
+const encryptKey = (key: string, salt: number) => {
+    const encodedData = btoa(key);
+    const saltValue = Math.floor(Math.random() * 10000000 * salt);
```

Comment:
the salt value must be constant, otherwise you'll never be able to decrypt the secret :)